king_phisher/catalog.py

```diff
@@ -344,6 +348,52 @@ def from_url(cls, url, keys=None, encoding='utf-8'):
 		keys.verify_dict(data, signature_encoding='base64')
 		return cls(data, keys=keys)
 
+class CatalogManager(object):
+	"""
+	Base manager for handling multiple Catalogs
+	"""
+	def __init__(self, url_catalog=None):
+		self.catalogs = {}
+
+		if url_catalog:
+			self.add_catalog_url(url_catalog)
+
+	def catalog_ids(self):
+		"""
+		The key names of the catalogs in the manager
+		:return: the catalogs ids in CatalogManager
+		:rtype: list
+		"""
+		return self.catalogs.keys()
+
+	def get_repos(self, catalog_id):
+		"""
+		Returns a list of repositories from the requested catalog
+		:param str catalog_id: The name of the catalog in which to get names of repositories from
+		:return: list
+		"""
+		return [repository for repository in self.catalogs[catalog_id].repositories]
```

Comment:
`self.catalogs[catalog_id].repositories` is already going to be a tuple. Does this method need to convert it to a list to make it mutable? Even if it does, this should simply be `list(self.catalogs[catalog_id].repositories)`.